packages/sources/layer2-sequencer-health/src/config.ts

```diff
@@ -0,0 +1,38 @@
+import { Requester, util } from '@chainlink/ea-bootstrap'
+import { Config } from '@chainlink/types'
+
+export const NAME = 'LAYER2_SEQUENCER_HEALTH'
+
+// 3 minutes
+export const DEFAULT_DELTA_TIME = 3 * 60 * 1000
+
+export enum Networks {
+  Arbitrum = 'arbitrum',
+  Optimism = 'optimism',
+}
+
+export const RPC_ENDPOINTS = {
+  [Networks.Arbitrum]: 'https://arb1.arbitrum.io/rpc',
+  [Networks.Optimism]: 'https://mainnet.optimism.io',
+}
+
+export const HEALTH_ENDPOINTS = {
+  [Networks.Arbitrum]: {
+    endpoint: undefined,
+    responsePath: [],
+  },
+  [Networks.Optimism]: {
+    endpoint: 'https://mainnet-sequencer.optimism.io/health',
+    responsePath: ['healthy'],
+  },
+}
```

Comment:
Are these endpoints considered completely static? Should we add the ability to override these with ENV vars?

Specifically thinking about `https://arb1.arbitrum.io/rpc`. "arb1" implies there could be an "arb2". Can node ops switch if "arb1" goes down?

Could also be useful if they want to use testnet endpoints.

Reply:
That sounds better, I'll change it